mcs/class/System.XML/Dummy.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Xml.Schema;
+
+namespace System.Xml
```

Comment:
It's not different namespace but parent namespace which is fine if the sources use System based namespace which I think they do. I didn't try to build it myself